IndomitableIbises/Bangazon-Sprint-3
# Pzc edit profile form

## Description of Proposed Changes

Moved profile creation to register page (can revisit this decision, since it's not what we talked about in the beginning, but was recommended by Joe)
Adding profile view, form and template that includes appropriate information
Adding edit_profile view, form and template that allows for editing of appropriate information
- 

## Steps to Test
```
git fetch --al
git pull origin pzc-edit-profile-form
```
purge your database and migrations, and make migrations/db

don't worry about faker

1. register a new user
2. navigate to /profile
3. see current profile information
4. click edit (to navigate to /edit_profile
5. change information in the profile
6. click save
7. be redirected to /profile, see changes are saved

## Impacted Areas in Application
registration
profile page
edit profile page
